Tests/FakeBundle/Controller/FakeController.php

```diff
@@ -303,7 +303,7 @@ public function paginatorAction(Paginator $paginator)
     public function paginatorSimpleAction(Paginator $paginator)
     {
         return array(
-            'count' => count($paginator)
+            'count' => $paginator->getIterator()->count()
```

Comment:
It would be easier if `Paginator` implements [`Countable`](http://php.net/manual/en/class.countable.php), wouldn't it?